models.go

```diff
@@ -496,7 +496,7 @@ type RealmRepresentation struct {
 	UserFederationMappers               []interface{}     `json:"userFederationMappers,omitempty"`
 	UserFederationProviders             []interface{}     `json:"userFederationProviders,omitempty"`
 	UserManagedAccessAllowed            bool              `json:"userManagedAccessAllowed"`
-	Users                               []interface{}     `json:"users,omitempty"`
+	Users                               []User            `json:"users,omitempty"`
```

Comment:
I am not sure why `[]interface{}` does not take `[]User `. It should be like a wild card and take any array input.